src/views/SystemProfile/components/FundingAndBudget/index.tsx

```diff
@@ -14,100 +23,190 @@ import {
 } from 'components/shared/DescriptionGroup';
 import Divider from 'components/shared/Divider';
 import SectionWrapper from 'components/shared/SectionWrapper';
-import Tag from 'components/shared/Tag';
+import { BUDGET_ITEMS_COUNT_CAP } from 'constants/systemProfile';
 import useCheckResponsiveScreen from 'hooks/checkMobile';
 import { SystemProfileSubviewProps } from 'types/systemProfile';
+import formatDollars from 'utils/formatDollars';
 
 import './index.scss';
 
 const FundingAndBudget = ({ system }: SystemProfileSubviewProps) => {
   const { t } = useTranslation('systemProfile');
   const isMobile = useCheckResponsiveScreen('tablet');
+
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
+
+  let budgetItemsLeft = 0;
+  const capEnd = isExpanded ? undefined : BUDGET_ITEMS_COUNT_CAP;
+
+  if (system.budgets !== undefined) {
+    budgetItemsLeft = system.budgets?.length - BUDGET_ITEMS_COUNT_CAP;
+  }
+
   return (
     <>
-      <SectionWrapper className="padding-bottom-4">
+      <SectionWrapper>
         <h2 className="margin-top-0 margin-bottom-4">
           {t('singleSystem.fundingAndBudget.header')}
         </h2>
+        <Table bordered={false} fullWidth scrollable>
+          <thead>
+            <tr>
+              <th scope="col">
+                {t('singleSystem.fundingAndBudget.fiscalYear')}
+              </th>
+              <th scope="col">
+                {t('singleSystem.fundingAndBudget.actualSystemCost')}
+              </th>
+              <th scope="col">
+                {t('singleSystem.fundingAndBudget.budgetedSystemCost')}
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {system.budgetSystemCosts?.budgetActualCost
+              .toReversed() // Reversed to show years in descending order
```

Comment:
is it preferable to just have the BE return it in the order the UI wants?

Reply:
I moved the sorting to the BE (in the CEDAR sdk handler specifically).